save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/CpgView.kt

```diff
@@ -98,11 +109,31 @@ val cpgView: VFC = VFC {
                                     this.graph = graphology.MultiDirectedGraph
                                     graphEvents {
                                         shouldHideUnfocusedNodes = true
+                                        setSelectedNode = { newSelectedNodeName ->
+                                            setSelectedNodeName(
+                                                newSelectedNodeName.takeIf { it != selectedNodeName }
+                                            )
+                                        }
                                     }
                                     graphLoader {
                                         cpgGraph = cpgResult.cpgGraph
                                     }
                                 }
+                                div {
+                                    id = "collapse"
+                                    val show = selectedNodeName?.let { nodeName ->
```

Comment:
Can you add some button to close this additional information window? Something like the X in the top right?

Reply:
The window is closed when clicking on the same node once again. Do you think it might be better to add such a button?

Reply:
Can it happen that this window will appear on top of the node that was clicked on?

Reply:
It can, but the graph layout is moveable so there is no problem with that

Reply:
Added close button to the top of the window